### DISCUSSION TITLE
Sense attached TFT Display

### DISCUSSION THREAD

**Discussion by ams-hh** [Bodmer/TFT_eSPI#3508] [Q&A - General]
Good morning!
I know, that a ILI9341 TFT Display existence can be sensed by
clTFT.readcommand8(ILI9341_RDID4, 2) != 0

Is there something similar for a ST7735 TFT Display I can use to
sense, if the Display is attached or not?

Thank you!
Regards
amshh

**Comment by Basitadam** [comment]
Short answer is yes.
ST7735 commands can be used:
ST7735_RDID1   (0xDA)
ST7735_RDID2   (0xDB)
ST7735_RDID3   (0xDC)
ST7735_RDID4   (0xDD)
You can refer to datasheet for the response decoding
Just a reminder: You must configure and connect the  MISO/SDI pin at the MCU side to ST7735 SDO pin
BR,
Tim

**Comment by ams-hh** [comment]
Thank you Tim
... and thanks for the reminder.
The Display is running in my environment since months.
Just needed this last detail.
